src/main.rs

```diff
@@ -93,16 +87,13 @@ async fn store_file(filename: &str, bytes: Bytes, args: &Args) -> std::io::Resul
     Ok(())
 }
 
-async fn accept_form(
-    ContentLengthLimit(mut multipart): LimitedMultipart,
-    Extension(args): Extension<Arc<Args>>,
-) -> Redirect {
+async fn accept_form(Extension(args): Extension<Arc<Args>>, mut multipart: Multipart) -> Redirect {
     // Store all files in the multipart body in the file system
     while let Some(field) = multipart.next_field().await.unwrap() {
         let file_name = field.file_name().unwrap().to_string();
         if let Err(e) = store_file(&file_name, field.bytes().await.unwrap(), &args).await {
             error!("Failed to upload file {}: {:?}", file_name, e);
         }
     }
-    Redirect::to("/".parse().unwrap())
+    Redirect::to("/")
```

Comment:
See https://github.com/tokio-rs/axum/blob/main/axum/CHANGELOG.md#050-31-march-2022

breaking: Redirect::{to, temporary, permanent} now accept &str instead of Uri (https://github.com/tokio-rs/axum/pull/889)